civihr_employee_portal/js/reports.js

```diff
@@ -431,13 +431,14 @@
    * for the leave reports when the filter values does not have a
    * query parameter appended.
    *
-   * @returns string {String}
+   * @param  {String} filterValues
+   *
+   * @return {String}
    */
   HRReport.prototype.processFilterValues = function (filterValues) {
-    var that = this;
-    //On load of page the form values are not appended to query string.
-    if (filterValues === '?' && that.reportName === 'leave_and_absence') {
-      return that.getDefaultFilterQueryForLeaveReport()
+    // On load of page the form values are not appended to query string.
```

Comment:
Now because we reduced the code, the comment can actually go to the function description.